build/version.sh

```diff
@@ -0,0 +1,174 @@
+#!/bin/bash
```

Comment:
Does this come from upstream somewhere? I don't want to vendor k8s/k8s for it, but maybe it's something we can push on to be in the infra repo.

Reply:
Yes, and it would make a good candidate for the infra repo. I'll modify the commit to be more explicit about the source.